DB/models.py

```diff
@@ -0,0 +1,47 @@
+# models.py
+import datetime
+
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, create_engine
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class User(Base):
+    """
+    People who upload files for explanation.
+    """
+    __tablename__ = 'users'
+
+    id = Column(Integer, primary_key=True)
+    email = Column(String, unique=True, nullable=False)
+
+    uploads = relationship("Upload", back_populates="user", cascade='all, delete-orphan')
+
+    # def __init__(self):
+
+
+class Upload(Base):
+    """
+    Files uploaded by people, with metadata related to their processing.
```

Comment:
By people is a tricky statement. either write users or dont write at all, because you cant know what changes your code will have in the future